src/styleguide/routes/r-picture.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <l-default>
+    <e-picture :sizes="sizes"
+               srcset="https://images.pexels.com/photos/10313905/pexels-photo-10313905.jpeg"
+               fallback="https://images.pexels.com/photos/9074921/pexels-photo-9074921.jpeg"
+               width="300"
+               height="1500"
+               alt="Image description"
+    />
+  </l-default>
+</template>
+
+<script lang="ts">
+  import { defineComponent } from 'vue';
+  import lDefault from '@/components/l-default.vue';
+  import ePicture from '@/components/e-picture.vue';
+
+  // interface ISetup {}
+  interface IData {
+    sizes: { [key: number | string]: number};
+  }
```

Comment:
todo: we should define the type/interface on the e-picture component instead, so we can re-use it across the application.